### PR TITLE
Kallax migrations documentation

### PR DESCRIPTION
**Pull request by erizocosmico**
Just last commit.

### CODE REVIEW

**Comment by ajnavarro** [comment]
This PR has code that is also on #147 . #147 should be merge first?

**Comment by erizocosmico** [comment]
Yep, #147 should be merged first.

File: README.md [hunk 1]
```diff
@@ -603,6 +604,95 @@ store.Transaction(func(s *UserStore) error {
 * `time.Time` and `url.URL` need to be used as is. That is, you can not use a type `Foo` being `type Foo time.Time`. `time.Time` and `url.URL` are types that are treated in a special way, if you do that, it would be the same as saying `type Foo struct { ... }` and kallax would no longer be able to identify the correct type.
 * Multidimensional arrays or slices are **not supported** except inside a JSON field.
 
+## Migrations
+
+Kallax can generate migrations for your schema automatically, if you want to. It is a process completely separated from the model generation, so it does not force you to generate your migrations using kallax.
+
+Sometimes, kallax won't be able to infer a type or you will want a specific column type for a field. You can specify so with the `sqltype` struct tag on a field.
```
Comment (Serabe) [inline README.md]: Example.

File: README.md [hunk 2]
```diff
@@ -603,6 +604,95 @@ store.Transaction(func(s *UserStore) error {
...
+
+These are the flags available for `up` and `down`:
+
+| Name | Description | Default |
+| --- | --- | --- |
+| `--dir` or `-d` | directory where your migrations are stored | `./migrations` |
+| `--dsn` | database connection string | required |
+| `--steps` or `-s` | maximum number of migrations to run | `0` |
+| `--all` | migrate all the way up (only available for `up` | 
+| `--version` or `-v` | final version of the database we want after running the migration. The version is the timestampt value at the beginning of migration files | `0` |
```
Comment (Serabe) [inline README.md]: s/timestampt/timestamp/

File: README.md (hunk 1 above)
Comment (erizocosmico) [inline README.md]: Fair point

**Comment by erizocosmico** [comment]
Done @Serabe

**Pull request merged**